.gitignore

```diff
@@ -37,3 +37,4 @@ node_modules
 Goopfile.lock
 venv/
 rultor_secrets.sh
+.vscode
```

Comment:
Move this out to a separate PR, or create a newcomer issue about it (for every repository...)?